### DISCUSSION TITLE
FEAT-REQUEST : Browsing files from navbar of code editor

### DISCUSSION THREAD

**Discussion by pankaj-2503** [processing/p5.js-web-editor#2874] [General]
### Increasing Access

This could be helpful in accessing  files that you are working more frequently.

### Feature request details

If  browsing file from the top of ide  could be possible then it would be helpful for accessing files .
As it would give more flexibility to work on different files . 

#  How it should looks like :
![image](https://github.com/processing/p5.js-web-editor/assets/90036864/f4374c94-3b1c-4610-ab9c-b68f01cd3682)

**Comment by welcome** [comment]
Welcome! 👋 Thanks for opening your first issue here! And to ensure the community is able to respond to your issue, be sure to follow the issue template if you haven't already.

**Comment by lindapaiste** [comment]
Can you please be more clear about what you are proposing?

**Comment by aakash2330** [comment]
I think he means like having different tabs (buffers) for opened files .

**Comment by rahulkhattri0** [comment]
This seems to be a nice feature

**Comment by aakash2330** [comment]
i'd like to work on it if we're moving forward with this @lindapaiste

**Comment by raclim** [comment]
Since this is a feature request that seems like it might need more context and design, I'm going to move this to Discussions!
